### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by smialko**
**SUMMARY** 
- it's sign-in/sign-up work in progress (!),
- there are views for creating an account and logging in, 
- you can create an account, the app sends info to API but nothing happens (only failure/success info Is printed in the debugger), 
- you can log in, the app contacts with the web but nothing then happens (only failure/success info Is printed in the debugger), 

**TO DO**
- saving data to user defaults,
- log out, 
- redirecting to specific views after signup/sign in,
- cleanup/ styling.

### CODE REVIEW

**Comment by anna1901** [comment]
Those are just some minor thoughts on the views. It would be great to go through the code of AuthorizationAPI together.

File: AirCasting/Auth/CreateAccountView.swift [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                .foregroundColor(.accentColor)
+            Text("to record and map your envitonment")
+                .font(Font.muli(size: 16))
+                .foregroundColor(.aircastingGray)
+        }
+    }
+    
+    var emailTextfield: some View {
+        createTextfield(placeholder: "Email",
+                        binding: $email)
```
[the remark this answers is not in the document]
Comment (smialko) [reply AirCasting/Auth/CreateAccountView.swift:50]: Good idea :D

File: AirCasting/Auth/CreateAccountView.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    signinButton
+                }
+            }
+            .padding()
+            .navigationBarHidden(true)
+        }
+    }
+    
+    var titleLabel: some View {
+        VStack(alignment: .leading, spacing: 15) {
```
[the remark this answers is not in the document]
Comment (smialko) [reply AirCasting/Auth/CreateAccountView.swift:37]: Yep, but it's like it's supposed to be, according to the designs  :) 
https://projects.invisionapp.com/d/main#/console/19745972/413017519/preview

File: AirCasting/Auth/SignInView.swift [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+struct SignInView: View {
+    
+    @State private var username: String = ""
+    @State private var password: String = ""
+    @State private var task: Any?
+    
+    var body: some View {
+            VStack(spacing: 40) {
+                titleLabel
```
[the remark this answers is not in the document]
Comment (smialko) [reply AirCasting/Auth/SignInView.swift:18]: Same here

**Pull request merged**